src/elements/ButtonBlock/ButtonBlock.tsx

```diff
@@ -0,0 +1,30 @@
+import { ButtonBlockNode } from '@prezly/story-content-format';
+import classNames from 'classnames';
+
+import './ButtonBlockNode.scss';
+
+interface ButtonNodeProps {
+    node: ButtonBlockNode;
+}
+
+export function ButtonBlock({ node }: ButtonNodeProps) {
+    const { href, label, layout, new_tab: newTab, variant } = node;
+
+    return (
+        <a
+            className={classNames('prezly-slate-button-block', {
+                'prezly-slate-button-block--disabled': !href,
```

Comment:
We don't want to render buttons without `href`.
See https://linear.app/prezly/issue/DEV-11229/design-buttons-in-storyless-campaigns#comment-701415b7

Reply:
Updated